crates/evm/coverage/src/analysis.rs

```diff
@@ -424,12 +425,20 @@ impl<'a> ContractVisitor<'a> {
                             let branch_id = self.branch_id;
                             self.branch_id += 1;
                             self.push_item(CoverageItem {
-                                kind: CoverageItemKind::Branch { branch_id, path_id: 0 },
+                                kind: CoverageItemKind::Branch {
+                                    branch_id,
+                                    path_id: 0,
+                                    is_first_opcode: false,
+                                },
                                 loc: self.source_location_for(&node.src),
                                 hits: 0,
                             });
                             self.push_item(CoverageItem {
-                                kind: CoverageItemKind::Branch { branch_id, path_id: 1 },
+                                kind: CoverageItemKind::Branch {
```

Comment:
realized that most (all?) calls to `push_item` set `loc: self.source_location_for(src), hits: 0`. Do you mind making a PR that adds `push_item_kind` that accepts `kind` and `src` to dedup this?

Reply:
pls check https://github.com/foundry-rs/foundry/pull/8616